server/test/lib/system/system.getGladysBasePath.test.js

```diff
@@ -49,6 +49,14 @@ describe('system.getGladysBasePath', () => {
       basePathOnContainer: '/var/lib/gladysassistant',
     });
   });
+  it('should return default basePath because failed to get mount', async () => {
```

Comment:
Same, not sure we want to soft fail for that, in this PR

Reply:
Ok, I have removed this change